doc/users/next_whats_new/svg_id_rc.rst

```diff
@@ -0,0 +1,31 @@
+``svg.id`` rcParam
+~~~~~~~~~~~~~~~~~~
+The value of this new rcParam controls whether the top-level ``<svg>`` tag
+contains an ``id`` attribute and what its value is. When set to ``None`` (the
```

Comment:
```suggestion
contains an :rc:`svg.id` attribute and what its value is. When set to ``None`` (the
```

to use our custom role.

Reply:
No. Here, `id` is the attibute ins `<svg>`.

Reply:
I took the liberty of rephrasing to make it more concise.